keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/ClusterSpec.kt

```diff
@@ -138,13 +135,17 @@ data class ClusterSpec(
     val regions: Set<ClusterRegion>
   )
 
+  operator fun Locations.get(region: String) =
+    regions.first { it.region == region }
+
   data class ClusterRegion(
     val region: String,
     val subnet: String,
     /**
      * If empty this implies the cluster should use all availability zones.
      */
-    val availabilityZones: Set<String> = emptySet()
+    val availabilityZones: Set<String> = emptySet(),
+    val image: VirtualMachineImage? = null
   )
```

Comment:
`ClusterRegion` now has an optional `image` property. The 99% case will have this getting resolved using the `ImageProvider`, but if you really want to you could stick a particular AMI id on here.